.github/workflows/tests.yml

```diff
@@ -8,28 +8,27 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Ariadne in it's readme states 3.7+, should this follow? Even if you are dropping 3.8 here it should be loudly stated in the release notes.

Reply:
Ariadne GraphQL Modules never supported 3.7 due to difference in bound methods handling in Python itself that landed in py3.8.